### PR TITLE
[stress-tests] add service ALOC connectivity stress test

### PR DESCRIPTION
**Pull request by simonlingoogle**
Test setup:
- Nodes pings the BR by it's service ALOC and measure the connectivity
- Topology
  - Router x20
  - FED x10
  - MED x10
  - SED x10
- Fault Injections:
  - Nodes are constantly moving
  - Nodes fail for 60s in every 600s
  - Packet Loss Ratio set to 0.2
- Pass Criteria
  - Max Delay (of all nodes) <= 3600s

### CODE REVIEW

**Comment by wgtdkp** [comment]
> Test setup:
> 
> * Nodes pings the BR by it's service ALOC and measure the connectivity
> * Topology
>   
>   * Router x20
>   * FED x10
>   * MED x10
>   * SED x10
> * Fault Injections:
>   
>   * Nodes are constantly moving
>   * Nodes fail for 60s in every 600s
>   * Packet Loss Ratio set to 0.2
> * Pass Criteria
>   
>   * Max Delay (of all nodes) <= 3600s

@simonlingoogle I'd like this documentation be also at the head of the test script file. So the reader can figure out the test easier. Thoughts?

**Comment by simonlingoogle** [comment]
> > Test setup:
> > 
> > * Nodes pings the BR by it's service ALOC and measure the connectivity
> > * Topology
> >   
> >   * Router x20
> >   * FED x10
> >   * MED x10
> >   * SED x10
> > * Fault Injections:
> >   
> >   * Nodes are constantly moving
> >   * Nodes fail for 60s in every 600s
> >   * Packet Loss Ratio set to 0.2
> > * Pass Criteria
> >   
> >   * Max Delay (of all nodes) <= 3600s
> 
> @simonlingoogle I'd like this documentation be also at the head of the test script file. So the reader can figure out the test easier. Thoughts?

added

**Review by wgtdkp — APPROVED** [review]
LGTM 👍

File: pylibs/stress_tests/errors.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# !/usr/bin/env python3
+# Copyright (c) 2020, The OTNS Authors.
```
Comment (jwhui) [inline pylibs/stress_tests/errors.py:2]:
```suggestion
# !/usr/bin/env python3
#
# Copyright (c) 2020, The OTNS Authors.
```

**Review by jwhui — COMMENTED** [review]
One very minor nit.

**Comment by jwhui** [comment]
@simonlingoogle , please help resolve conflicts.

**Comment by jwhui** [comment]
@simonlingoogle , another conflict after merging #92 :)

**Comment by simonlingoogle** [comment]
> @simonlingoogle , another conflict after merging #92 :)

resolved

**Review by jwhui — APPROVED** [review]

**Pull request merged**